src/containers/Home/Home.test.tsx

```diff
@@ -1,9 +1,69 @@
-import {render} from "@testing-library/react";
-import Home from "../Home/Home";
+import { render, waitFor } from "@testing-library/react";
+import App from "../App/App";
```

Comment:
With Home.test.tsx, we should only test files in the Home component

Reply:
I mean if you have a file `Home.test.tsx` you should only use it to test `Home/index.tsx`, you should not test `../App/App` in that